src/table/header-cell/th-element.tsx

```diff
@@ -42,13 +45,17 @@ export function TableThElement({
     getClassName: props => getStickyClassNames(styles, props),
   });
 
-  const mergedRef = useMergeRefs(stickyStyles.ref, cellRef);
+  const cellRefObject = useRef<HTMLTableCellElement>(null);
+  const mergedRef = useMergeRefs(stickyStyles.ref, cellRef, cellRefObject);
+  const { tabIndex: cellTabIndex } = useSingleTabStopNavigation(cellRefObject);
 
   return (
     <th
+      data-focus-id={`header-${String(columnId)}`}
```

Comment:
Why is this necessary?

Reply:
The data-focus-id is used to keep focus outlines in sync between header and sticky header. Previously it was only possible to focus on sortable headers, select-all checkbox and resize handles. Now it is also possible to focus on header cells themselves when there are no other focus targets inside.